### PR TITLE
T() macro 8

### PR DESCRIPTION
**Pull request by Dragorn421**
Sequel to #2064

Finishes up T() in code, besides whatever I may have missed and db_camera.c (which I'm not doing lol)

### CODE REVIEW

File: src/code/z_message.c [hunk 1]
```diff
@@ -3228,16 +3208,14 @@ void Message_DrawMain(PlayState* play, Gfx** p) {
                 if (msgCtx->stateTimer == 0) {
                     R_OCARINA_BUTTONS_YPOS_OFFSET = 1;
                     if (msgCtx->msgMode == MSGMODE_SONG_PLAYBACK_FAIL) {
-                        // "kokokokokoko"
-                        PRINTF("ここここここ\n");
+                        PRINTF(T("ここここここ\n", "kokokokokoko\n"));
```
Comment (cadmic) [inline src/code/z_message.c:3211]: probably should translate this ("here here here" apparently)

File: src/code/fault_gc.c [hunk 2]
```diff
@@ -674,12 +681,12 @@ void Fault_WaitForButtonCombo(void) {
...
-    // KeyWaitB (LRZ Up Down Up Down Left Left Right Right B A START)
+    // "KeyWaitB (LRZ Up Down Up Down Left Left Right Right B A START)"
     osSyncPrintf(
         VT_FGCOL(WHITE) "KeyWaitB (ＬＲＺ " VT_FGCOL(WHITE) "上" VT_FGCOL(YELLOW) "下 " VT_FGCOL(YELLOW) "上" VT_FGCOL(WHITE) "下 " VT_FGCOL(WHITE) "左" VT_FGCOL(
             YELLOW) "左 " VT_FGCOL(YELLOW) "右" VT_FGCOL(WHITE) "右 " VT_FGCOL(GREEN) "Ｂ" VT_FGCOL(BLUE) "Ａ" VT_FGCOL(RED) "START" VT_FGCOL(WHITE) ")" VT_RST
                                                                                                                                                      "\n");
-    // KeyWaitB'(LR Left Right START)
+    // "KeyWaitB'(LR Left Right START)"
     osSyncPrintf(VT_FGCOL(WHITE) "KeyWaitB'(ＬＲ左" VT_FGCOL(YELLOW) "右 +" VT_FGCOL(RED) "START" VT_FGCOL(
         WHITE) ")" VT_RST "\n");
```
[the remark this answers is not in the document]
Comment (Dragorn421) [reply src/code/fault_gc.c:691]: Here are the options for `KeyWaitB` (formatter moment):
```c
    osSyncPrintf(T(
        VT_FGCOL(WHITE) "KeyWaitB (ＬＲＺ " VT_FGCOL(WHITE) "上" VT_FGCOL(YELLOW) "下 " VT_FGCOL(YELLOW) "上" VT_FGCOL(WHITE) "下 " VT_FGCOL(WHITE) "左" VT_FGCOL(
            YELLOW) "左 " VT_FGCOL(YELLOW) "右" VT_FGCOL(WHITE) "右 " VT_FGCOL(GREEN) "Ｂ" VT_FGCOL(BLUE) "Ａ" VT_FGCOL(RED) "START" VT_FGCOL(WHITE) ")" VT_RST,
        VT_FGCOL(WHITE) "KeyWaitB (LRZ " VT_FGCOL(WHITE) "Up" VT_FGCOL(YELLOW) "Down " VT_FGCOL(YELLOW) "Up" VT_FGCOL(WHITE) "Down " VT_FGCOL(WHITE) "Left" VT_FGCOL(
            YELLOW) "Left " VT_FGCOL(YELLOW) "Right" VT_FGCOL(WHITE) "Right " VT_FGCOL(GREEN) "B" VT_FGCOL(BLUE) "A" VT_FGCOL(RED) "STAR"
                                                                                                                                   "T" VT_FGCOL(
                                                                                                                                       WHITE) ")" VT_RST) "\n");
```
or
```c
    osSyncPrintf(VT_FGCOL(WHITE) T("KeyWaitB (ＬＲＺ ", "KeyWaitB (LRZ ") VT_FGCOL(WHITE) T("上", "Up") VT_FGCOL(YELLOW)
                     T("下 ", "Down ") VT_FGCOL(YELLOW) T("上", "Up") VT_FGCOL(WHITE) T("下 ", "Down ") VT_FGCOL(WHITE)
                         T("左", "Left") VT_FGCOL(YELLOW) T("左 ", "Left ") VT_FGCOL(YELLOW) T("右", "Right")
                             VT_FGCOL(WHITE) T("右 ", "Right ") VT_FGCOL(GREEN) T("Ｂ", "B") VT_FGCOL(BLUE) T("Ａ", "A")
                                 VT_FGCOL(RED) "START" VT_FGCOL(WHITE) ")" VT_RST "\n");
```

[the remark this answers is not in the document]
Comment (fig02) [reply src/code/fault_gc.c:691]: tbh just ignore the colors in the translation imo
I get its not accurate, but along with outputting the translated text these T macros serve to make it easier to read the code as well. And that is too much of a mess

[the remark this answers is not in the document]
Comment (cadmic) [reply src/code/fault_gc.c:691]: I think I prefer option 2, it's more consistent with how other VT() + T() have been done, and the formatting is less offensive

[the remark this answers is not in the document]
Comment (Dragorn421) [reply src/code/fault_gc.c:691]: unfortunately the colors have meaning, e.g. yellow "Left" is C-Left while white "Left" is Dpad-Left
I'll push option 2 then

[the remark this answers is not in the document]
Comment (cadmic) [reply src/code/fault_gc.c:691]: maybe we could include both the T() macro and the comment in this case?

[the remark this answers is not in the document]
Comment (Dragorn421) [reply src/code/fault_gc.c]: pushed option 2 with more spaces and comments

**Review by cadmic — APPROVED** [review]

**Review by fig02 — APPROVED** [review]

**Pull request merged**